Naming convention (#1540)
nasa-gcn/gcn.nasa.gov: app/routes/docs.notices.producers.mdx

      <ProcessListHeading type="h3">
         Name Your Kafka Topics
      </ProcessListHeading>

>       The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>NoticesType</i></code>. The mission name should be lower case, while the Kafka topics should be in [CamelCase](https://en.wikipedia.org/wiki/Camel_case) format.

Author: ```suggestion
      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<I>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<I>instrument</i>.<I>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<I>swift</i>.<I>bat</i>.<I>alert</i></code>.
```

>       The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<I>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<I>instrument</i>.<I>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<I>swift</i>.<I>bat</i>.<I>alert</i></code>.

Reviewer: ```suggestion
      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<i>instrument</i>.<i>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<i>swift</i>.<i>bat</i>.<i>alert</i></code>.
```

      <ProcessListHeading type="h3">
         Name Your Kafka Topics
      </ProcessListHeading>
      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>NoticesType</i></code>. The mission name should be lower case, while the Kafka topics should be in [CamelCase](https://en.wikipedia.org/wiki/Camel_case) format.

>       Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>. 

Reviewer 2: We shouldn't link to wikipedia.  Also, what about gcn.notices.mission.instrument.NoticeType?  Instrument is optional. And NoticesType should be NoticeType

Author: Ok, other reference sites are worse and have advertisements. "instrument.NoticeType" extra instrument word is my mistake. 
I was trying to keep Notices as a consistent word for GCN Notices.

Reviewer: > We shouldn't link to wikipedia.

Why not?

Reviewer 2: It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites.  Anyone can change the content of wikipedia.

Reviewer: > It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites. Anyone can change the content of wikipedia.

Is there a link to that policy? I haven't heard of it before.

>       Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>. 

Reviewer: ```suggestion
      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>.
```
